app/views/publications/monthly_statistics/v2/show.html.erb

```diff
@@ -121,7 +121,7 @@
 
     <p class="govuk-body">These statistics cover applications for courses in England starting in the <%= @presenter.academic_year_name %> academic year. To allow for comparison statistics covering the <%= @presenter.current_cycle_name %> academic year are also included.</p>
 
-    <p class="govuk-body">New definitions and methodology have been introduced for the <%= @presenter.academic_year_name %> academic year because candidates can now submit applications to courses individually up to a maximum limit of 4 open applications at any one time. Previously candidates submitted an application form containing up to <%= ApplicationForm::MAXIMUM_NUMBER_OF_COURSE_CHOICES %> applications at the same time, all of which had to be processed before the candidate could submit another application form that cycle.</p>
+    <p class="govuk-body">New definitions and methodology have been introduced for the <%= @presenter.academic_year_name %> academic year because candidates can now submit applications to courses individually up to a maximum of 4 open applications at a time. Previously, candidates submitted one application form with up to <%= ApplicationForm::MAXIMUM_NUMBER_OF_COURSE_CHOICES %> course choices at the same time. All 4 course choices had to receive a decision before the candidate could submit another application form in the same recruitment cycle.</p>
```

Comment:
I know it's unrelated but do those 4's need to be replaced with `MAXIMUM_NUMBER_OF_COURSE_CHOICES`?

Reply:
Good spot, done.